### ISSUE TITLE
Fetch is not defined with Unsplash package for Nodejs development environment

### ISSUE THREAD

**Issue by bugb** [unsplash/unsplash-js#98]
Environment: Nodejs

I think the README.md file should be updated base on comment: https://github.com/unsplash/unsplash-js/issues/35#issuecomment-272625695 for more detail for Nodejs user.

And I make another solution here:
Step 1: `npm i --save unsplash-js node-fetch`
Step 2: Create sample js file like this:
```
const Unsplash = require("unsplash-js").default;
const fetch = require('node-fetch');
global.fetch =fetch;

const unsplash = new Unsplash({
  applicationId: "yourId",
  secret: "yourSecret"
});
unsplash.photos
  .getRandomPhoto()
  .then(res => res.json())
  .then(json => {
    console.dir(json);
  });

```
Step 3: This should be worked.

**Comment by Dayjo** [comment]
I do agree with this, a simple documentation improvement would have probably saved me a good 20/30 minutes.

**Comment by antonioallen** [comment]
@bugb you just saved me hours! Thank you! 👏

**Comment by bugb** [comment]
@antonioallen Glad if my comment can solve your problem.

I think the short term we can update the README.md file
Should I make a PR, but I do not know if my MR have any chance to got merged !?

And for long term the owner should consider to switch to another library like `axios`, it work great.

**Comment by BeKnowDo** [comment]
Life...saver @bugb

**Comment by Haccker3005** [comment]
After trying to figure it out for 45 minutes, I am so glad I found this! Thank you so much!

**Comment by Dayjo** [comment]
@bugb Definitely create a PR, doesn't look like anyone here has got around to doing it :) It'll help a bunch of folks.

**Comment by lukechesser** [comment]
Documented in #117 and will be released in #112
